Lateral Acceleration Error alert (Sensor Invalid)

nuwandavek opened the pull request:
Large Lateral Acceleration discrepancies (caused by shaking the device / device falling) trigger this alert.
![image](https://user-images.githubusercontent.com/1649262/181126832-518d3192-b3fa-4e4f-ab3c-6e9cae57a458.png)

nuwandavek committed:
sensor flag in paramsd

nuwandavek committed:
increase lateral acc err alert threshold

nuwandavek committed:
add invalid sensor alert from paramsd

nuwandavek committed:
update ref, bool before adding to msg

pd0wm reviewed (approved):
LGTM

What exactly is the process replay diff?

nuwandavek committed:
account for sign difference in locationd and paramsd

nuwandavek committed:
revert ref

nuwandavek commented:
> What exactly is the process replay diff?
This was a bug! I should have checked the ref properly, thanks for pointing out.

The offline paramsd was slightly different from the online one, and there is a sign difference between `locationd` and `paramsd` yaw rates. Fixed that and reverted the ref update. (and also made sure online and offline behaviour is the same).
